Automattic/wp-calypso
# Add `We’ll take it from here!` step when user tries to migrate a wpcom site

Closes #95293

## Proposed Changes

* Add a step to provide user info about the HE's next steps after the user provide information about their goals
*  Connect the step with the `migration` and `site-migration` flows, after the user submits the form

> [!NOTE]  
> There is another scenario we should display this screen It will be implemented as part of task #95293 but the content is already available on this PR. 

## Testing Instructions
* Go to `/setup/migration` flow
* Follow all steps until reach the credentials step
* Set a site that is already hosted on wordpress.com (E.g.  [any site].wpcomstaging.com)
* Set any goal option on the `Your site is already on WordPress.com`
* Check if you can see the new step as the image above (Please pay attention if the subtitle is the same as the screenshot because there is another variation that will be connected to another scenario in a further PR.
* Check the links

![image](https://github.com/user-attachments/assets/d08dd2dc-df0e-422d-bf99-7f342df3a0c6)

 

## Pre-merge Checklist
